extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts

```diff
@@ -57,7 +57,10 @@ export class PageViewPerformanceManager {
         if (navigationTiming || timing) {
             if (navigationTiming) {
                 total = navigationTiming.duration;
-                network = total - dateTimeUtilsDuration(navigationTiming.connectEnd, navigationTiming.loadEventEnd);
+                let afterNetwork = dateTimeUtilsDuration(navigationTiming.connectEnd, navigationTiming.loadEventEnd);
+                if (!isNotNullOrUndefined(afterNetwork)) {
```

Comment:
This still won't populate the values... As per the bug details, startTime is always zero so dateTimeUtilsDuration() will always return null...

Reply:
Yeah so I avoid using startTime. Instead of checking the difference of `startTime` between `connectEnd` (which is always null), the network time is `total` minus `afterNetwork`. No `startTime` involved here. Did I miss something... 
![image](https://user-images.githubusercontent.com/20292261/114755876-fcb8bb00-9d0e-11eb-827f-15ce729b72ec.png)

Reply:
Oh I guess this is comparing my first commit with my second commit... I added a null check for afterNetwork, making sure `total - afterNetwork` doesn't make network = total (total - null = total). If afterNetwork is null, network is still 0.

Reply:
Oh, I missed that part I was just looking how it was getting set and not the value.
Using afterNetwork - total, I think is also not quite the correct approach as it might potentially include additional time on some browsers, I was thinking this change should be something a bit simplier.
```
if (navigationTiming.startTime === 0) {
   network = navigationTiming.connectEnd;
} else {
  // Support older browsers where the startTime is not zero
  network = dateTimeUtilsDuration(navigationTiming.startTime, navigationTiming.connectEnd);
}
```